crates/notion-core/src/distro/package.rs

```diff
@@ -542,6 +620,23 @@ impl UserTool {
     }
 }
 
+fn bin_full_path<P, E>(
```

Comment:
This function is actually used in 2 places: One line 581, and immediately above the Regex stuff on line 453. That said, I did seriously consider inlining it in both places, because the function call is almost as long, if not longer, than the fully written out code. It was one of those spots where I felt like the two options were almost the same, and I'm willing to be convinced either way.

Reply:
Ah, I missed one of those somehow – editor fail. Totally your call, then!